src/doc/unstable-book/src/compiler-flags/fmt-debug.md

```diff
@@ -0,0 +1,15 @@
+# `fmt-debug`
```

Comment:
Missing the entry in the [Checking conditional configurations - rustc book](https://github.com/rust-lang/rust/blob/a971212545766fdfe0dd68e5d968133f79944a19/src/doc/rustc/src/check-cfg.md?plain=1#L102-L135): `src/doc/rustc/src/check-cfg.md`

*(don't forget to also update the date)*

Reply:
It's there: https://github.com/rust-lang/rust/blob/38659fd39ab70468759adb8aebef6ab846ee5cfa/src/doc/rustc/src/check-cfg.md?plain=1#L110